package.json

```diff
@@ -326,8 +343,9 @@
         "@types/micromatch": "^4.0.9",
         "@types/mocha": "^10.0.7",
         "@types/node": "~20.14.10",
+        "@types/semver": "^7.5.8",
         "@types/validate-npm-package-name": "^4.0.2",
-        "@types/vscode": "1.89.0",
```

Comment:
I would like to update "@types/vscode" and "engines.vscode" to 1.89.0, this extension only support latest 2 version of vscode.